db/migrate/20221122095639_create_posts.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.references :user, null: false, foreign_key: true
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
https://github.com/Josphat205/rails-blog/blob/e49d14bb26f5baccd82a3191b618f2b384c99b8d/db/migrate/20221122094053_create_users.rb#L1-L11

- [ ] Kindly add a `posts_counter` column to the `Users` table, as per the following [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).

Reply:
The posts counter is already exist, i added using a different migration